### PR TITLE
[IMP] Badges and translation config

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +11,7 @@ This project aim to deal with modules related to manage Odoo server environment
  - Manage configuration depending on environment (devs, test, prod,..)
  - Keep the security on update
  - Manage email settings
- -...
+
+Translation Status
+------------------
+[![Transifex Status](https://www.transifex.com/projects/p/OCA-server-tools-8-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-server-tools-8-0)
```
Comment (moylop260) [inline README.md:17]: Page not found

**Comment by moylop260** [comment]
<s>just a comment: link translation not found out of this </s>
:+1:

File: README.md (hunk 1 above)
Comment (pedrobaeza) [inline README.md:17]: This happens until the first commit, so we should check it knowing that it will be this way :wink:

Comment (moylop260) [inline README.md:17]: Oh this pr added transifex in travis.yml. Great!
My bad!

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: J6Zr34aVnjXFRwIamLmnlsN42t8ki2syAvQ26xJ+uOuJUkDLjSDnvrkFtss/Wn1watIERSrw/T8TrAZ0GxhUQ7B8jq7EZFmewobhyBpQVBR58S2clcGupYXgLUk9GwYfbxb7uAgIM8YOlnIteE69gMXJubzBx5pxEHOayUPbc/s=
```
Comment (yvaucher) [inline .travis.yml]: I have 'OCA/server-tools': 'Z06mZCN+Hm3myqHSOZpOOk1pd4oq1epAWZv6m9OX2bTNHbhyOVOGK6JWWsnDm/3DUCN1ZeLtSGOl9bvQfMa8ahQHA80MkLL16YlTvQV59Lh+L2gAYmxX+ogJCJgeQSVAXlGLscgkADCu/HzDlmatrDeROMtULn5i23j2qcyUNyM=',

On https://github.com/OCA/maintainer-quality-tools/issues/194

We definitively needs to do that with a script for this I guess.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**